### ISSUE TITLE
Configuration to disable nested project search

### ISSUE THREAD

**Issue by llgcode** [redhat-developer/vscode-java#188]
I would be interested to have an option to disable searching for projects in folders when there's already one project in the parent folder or maybe more helpful provides a configuration file/property that list the projects paths that I want to be in the workspace even if these projects are not in the current open folder.

**Comment by gorkem** [comment]
Do you still want to see those directories. Asking if it makes sense to use the global `files.exclude` setting or not?

**Comment by llgcode** [comment]
Yes in some case I want to view these directories. But it's a good option to not browse folders that are excluded `files.exclude`.

**Comment by davidbruce** [comment]
When can we expect the enhancement from #255 to be released?

**Comment by fbricon** [comment]
Next release will be end of July. In the meantime you can try the latest java-*.vsix from http://download.jboss.org/jbosstools/jdt.ls/staging/

**Comment by davidbruce** [comment]
It looks like the latest build on staging doesn't have that PR.  Thank you though, I'll be patient for the release.  :)

**Comment by fbricon** [comment]
@davidbruce right, looks like our vscode build is broken and the staging area hasn't been updated in a while.

cc @snjeza, @gorkem

**Comment by snjeza** [comment]
I can't reproduce the issue locally. Where is our vscode build?

**Comment by fbricon** [comment]
@davidbruce http://download.jboss.org/jbosstools/jdt.ls/staging/ now contains the latest vsix builds

**Comment by llgcode** [comment]
I've configured "java.import.exclusions": [ "**/data" ]. "data" folder contains project that I don't want to be loaded but when java extension load it generates a bin folder on projects that are in data folder.

**Comment by MontyBlenheim** [comment]
Has there been any progress on this? I have a large codebase that I'm trying to create a VSCode workspace for, but I get lots of errors from projects in subdirectories. It would be great if I could exclude these projects and only have the one in the root directory be used.

**Comment by llgcode** [comment]
Hi @MontyBlenheim ,
for my part I use this setting to exclude java project:
```json
"java.import.exclusions": [
        "**/node_modules/**",
        "**/.metadata/**",
        "**/archetype-resources/**",
        "**/META-INF/maven/**",
        "**/newfoldertoexclude/**"
]
```
